### PR TITLE
add plot of pvlive 

### LINKED ISSUE #49: Add PVlive intra day vs day-after
## Detailed Description
Would be good to get the metrics for PVlive intraday

## Context
Good to compare PVlive intra day and PVlive day -after

## Possible Implementation
Could go on the 'Quartz Solar MAE with RMSE for Comparision`

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

Add PVlive intraday to dat after metrics

Fixes #49 

Screenshot 2023-08-29 at 15 47 54

## How Has This Been Tested?

ran locally

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by rachel-labri-tipton — CHANGES_REQUESTED** [review]
This looks good, @peterdudfield. It could be useful to change the chart title on `line 401`: ex. `Quartz Solar and PVLive MAE with RMSE for Comparison` or something referring to PVLive.

**Comment by peterdudfield** [comment]
> This looks good, @peterdudfield. It could be useful to change the chart title on `line 401`: ex. `Quartz Solar and PVLive MAE with RMSE for Comparison` or something referring to PVLive.

Thanks @rachel-labri-tipton for this. Have updated

**Comment by peterdudfield** [comment]
Screenshot 2023-08-29 at 17 33 07

File: src/main.py [hunk 1]
```diff
@@ -398,7 +398,7 @@ def metric_page():
     # comparing MAE and RMSE
     fig6 = go.Figure(
         layout=go.Layout(
-            title=go.layout.Title(text="Quartz Solar MAE with RMSE for Comparison"),
+            title=go.layout.Title(text="Quartz Solar and PVlive MAE with RMSE for Comparison"),
```
Comment (rachel-labri-tipton) [inline src/main.py:402]: looks good!

File: src/main.py [hunk 2]
```diff
@@ -439,8 +439,8 @@ def metric_page():
     )
 
     fig6.update_layout(yaxis_range=[0, MAE_LIMIT_DEFAULT])
-    st.plotly_chart(fig6, theme="streamlit")\
-
+    st.plotly_chart(fig6, theme="streamlit")
+    st.text("PV live if the different between the intraday and day after PV Live values.")
```
Comment (rachel-labri-tipton) [inline src/main.py]: useful information. couple typos in here that I'll clean up.

**Review by rachel-labri-tipton — APPROVED** [review]

File: src/main.py (hunk 2 above)
Comment (peterdudfield) [reply src/main.py]: I'm happy to fix them

**Pull request merged**